src/Component/Calculator.js

```diff
@@ -1,47 +1,70 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
+    this.eventHandler = this.eventHandler.bind(this);
     this.state = {
+      total: '',
+      next: '',
+      operation: '',
     };
   }
 
+  eventHandler(newState) {
+    if (newState.total != null) {
+      this.setState({ total: newState.total });
+    }
+
+    this.setState({ next: newState.next });
+
+    if (newState.operation != null) {
+      this.setState({ operation: newState.operation });
+    }
+  }
```

Comment:
- [ ] As stated by the previous reviewer, your app is misbehaving. You should deal with the bugs and errors with proper return values and error handling. 
![chrome-capture (2)](https://user-images.githubusercontent.com/64894927/129414147-ddee0d27-b743-4d6f-8990-e39c0bfc0115.gif)
![chrome-capture (3)](https://user-images.githubusercontent.com/64894927/129414395-dce498e3-9b2a-4a08-8a73-e569d93cb74c.gif)
- [ ] The `+` operator is not responding at all. 
- [ ] Kindly fix these bugs. :pray: